### DISCUSSION TITLE
Comment exceptions

### DISCUSSION THREAD

**Discussion by waldo1001** [microsoft/alguidelines#157] [BC Best Practices]
They idea of this guidelines is: 

when you have to implement code that clear is doubtful, that you always put in a comment to explain why you do what you do.

Example: https://twitter.com/StefanMaron/status/1504723518436556807?s=20&t=4dpkoXoQL1B9KiAtFA7Ncw

Other example: any Commit ;-)

**Comment by StefanMaron** [comment]
Other Example: Write to a FlowField:
https://github.com/StefanMaron/BusinessCentral.LinterCop/wiki/LC0017
